### PR TITLE
refactor(ui): make chat session position fixed on scroll

### PR DESCRIPTION
**Pull request by liangfung**
https://jam.dev/c/ad6b2539-f2e5-45c0-bd25-101fd00bfe7e

### CODE REVIEW

File: ee/tabby-ui/components/demo-banner.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ export function DemoBanner() {
   return (
     <div
       className={cn(
-        'flex items-center justify-between bg-primary px-4 text-primary-foreground transition-all md:px-5',
+        'flex items-center justify-between bg-primary px-4 text-primary-foreground transition-all md:px-5 sticky top-0 z-10',
```
Comment (liangfung) [inline ee/tabby-ui/components/demo-banner.tsx]: @wwayne Is it fine to set this to sticky positioning?

Comment (wwayne) [reply ee/tabby-ui/components/demo-banner.tsx]: I feel sticky might not fit here. Let me update the PR for your review later

Comment (wwayne) [reply ee/tabby-ui/components/demo-banner.tsx]: Updated, please take a check https://jam.dev/c/38b5581c-d36a-4279-a547-31eb1253955b

**Review by wwayne — APPROVED** [review]

File: ee/tabby-ui/components/demo-banner.tsx (hunk 1 above)
Comment (liangfung) [reply ee/tabby-ui/components/demo-banner.tsx]: Using this layout will keep the `<ButtonScrollToBottom />` invisible.

**Comment by wwayne** [comment]
looks the scroll button has incompatible issue, it need to be re-consider for multiple cases, hold the PR for the moment

**Review by wwayne — APPROVED** [review]

**Comment by wwayne** [comment]
Add the button back https://jam.dev/c/f7bb1f7d-1243-496a-b789-d1d2b967e418
@liangfung please take a checking, thanks

**Comment by wsxiaoys** [comment]
Please resolve conflicts

**Pull request merged**